### PR TITLE
chore: change networks structure to match chain-registry's

### PR DESCRIPTION
**Pull request by jlehtimaki**
This PR will change  Archway Networks repository structure to match `chain-registry`s one.

`testnets` now hold all the testnets that archway has, where `archwaytestnet` being always the `latest` one
`testnets/_IBC` holds testnet IBC paths
`archway-1` was renamed to `archway` to correctly represent the network
`_IBC` holds all the IBC paths for `mainnet`

### CODE REVIEW

**Comment by aelesbao** [comment]
Questions:

1. Why do we need to keep old testnet files? If someone wants to consult this information, they can always refer to the git history.
2. Can we name it `archway-testnet` instead? 😎

File: .github/workflows/ibc-path-validation.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
           list-files: shell
           filters: |
             ibc:
-              - added|modified: '_IBC/*.json'
+              - added|modified: '**/_IBC/*.json'
```
Comment (kayano) [inline .github/workflows/ibc-path-validation.yaml:22]: remove maybe also
https://github.com/archway-network/networks/blob/main/.github/workflows/ibc-path-validation.yaml#L29-L33

Comment (jlehtimaki) [reply .github/workflows/ibc-path-validation.yaml:22]: Why those need to be removed?

Comment (kayano) [reply .github/workflows/ibc-path-validation.yaml:22]: The pipeline does sparse checkout and testnets dir is not on dirs list for the checkout

**Review by kayano — APPROVED** [review]

**Review by eliasmpw — APPROVED** [review]

**Pull request merged**